backend/muster-roll-service/src/main/java/org/egov/web/models/AttendanceLog.java

```diff
@@ -0,0 +1,67 @@
+package org.egov.web.models;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import digit.models.coremodels.AuditDetails;
+//import digit.models.coremodels.Document;
+import digit.models.coremodels.Document;
+import lombok.*;
+import org.springframework.validation.annotation.Validated;
+
+import javax.validation.Valid;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+/**
+ * AttendanceLog
+ */
+@Validated
+@javax.annotation.Generated(value = "org.egov.codegen.SpringBootCodegen", date = "2022-11-14T14:44:21.051+05:30")
+
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+public class AttendanceLog {
```

Comment:
Make the corresponding change to the API contract, if not done already.

Reply:
Updated the API contract. Thanks